lib/eventbrite/venue.rb

```diff
@@ -0,0 +1,35 @@
+class Eventbrite
+  class Venue
+    def initialize(details)
+      @details = details
+    end
+
+    def name
+      details["name"]
+    end
+
+    def address
+      [
+        street_and_city,
+        address_details["postal_code"]
+      ].reject(&:empty?).join(" ")
+    end
+
+    private
+
+    attr_reader :details
+
+    def street_and_city
+      [
+        address_details["address_1"],
+        address_details["address_2"],
+        address_details["city"],
+        address_details["region"],
+      ].compact.reject(&:empty?).join(", ")
+    end
+
+    def address_details
+      details["address"]
```

Comment:
Is there any chance this will be null which would cause the `street_and_city` calls to fail.